docs/user/FAQ.rst

```diff
@@ -110,3 +110,43 @@ dictionary. Below, there is example code on how to achieve this:
     # for each sample so that it can be weighted
     net = MyNet(MyModule, ..., criterion__reduce=False)
     net.fit(X, y)
+
+
+I already split my data into training and validation sets, how can I use them?
+------------------------------------------------------------------------------
+
+If you have already defined a training and validation datasets that are
+subclasses of :class:`~torch.utils.data.Dataset`, you can use
+:func:`~skorch.helpers.predefined_split` to train your network:
```

Comment:
Somehow the sentence "you can use `predefined_split()` to train your network" does not sound right to me. We are not using this function to train the network. We are using this function to attach to the network a fixed validation dataset that is independent on the training data passed to the `fit()` method.

Reply:
I see what you mean. I rephrased the sentence to try to make it more clear.